mis_builder/models/aep.py

```diff
@@ -169,7 +170,14 @@ def _parse_match_object(self, mo):
         # move line domain
         if ml_domain:
             assert ml_domain[0] == "[" and ml_domain[-1] == "]"
-            ml_domain = tuple(safe_eval(ml_domain, domain_eval_context))
+            components = safe_eval(ml_domain, domain_eval_context)
+            ml_domain = []
+            for component in components:
+                left, op, right = component
+                if isinstance(right, list):
+                    right = tuple(right)
```

Comment:
Is there not an Odoo function somewhere that implement that kind of logic?

Reply:
I also thought I've seen one, but haven't found any actually 🤷‍♂️